HW.11/HW.11.Task1/Song.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace HW._11.Task1
+{
+    public class Song : ISongService
+    {
+        public string Title { get; set; }
+        public double Minutes { get; set; }
+        public string Composer { get; set; }
+        public int AlbumYear { get; set; }
+        public Genre Genre { get; set; }
+        public GenreFlags GenreFlags { get; }
+
+        public Song(string title)
+        {
+            Title = title;
+        }
+        public Song(string title, double minutes, string composer, int albumYear, GenreFlags genre)
+        {
+            Title = title;
+            Minutes = minutes;
+            Composer = composer;
+            AlbumYear = albumYear;
+            GenreFlags = genre;
+        }
+        public override string ToString()
+        {
+            return $"Title - {Title}, Minutes - {Minutes}, AlbumYear - {AlbumYear}, Genre - {GenreFlags.ToString()}.";
+        }
+
+        public void GetSongsByGenre(List<Song> songs, string userGenre)
```

Comment:
the main idea was to put enum instead of `string userGenre`
using `[Flags]` attribute allows putting more than one value to one parameter, for instance

`GenreFlags genre = GenreFlags.Pop | GenreFlags.Rock | GenreFlags.Folk`;